docs/metrics/object-detection.md

```diff
@@ -0,0 +1,121 @@
+# Object Detection
+
+## Overview of Object Detection
+
+Object detection is a widely applied computer vision task. Object detection aims to locate and identify objects within
+an image or video, typically with bounding boxes.
+
+<figure markdown>
+  ![COCO_val2014_000000023294](../assets/images/metrics-tasks-od-COCO_val2014_000000023294_annotated.png)
+  <figcaption markdown>Person detection on an [MS COCO](https://paperswithcode.com/dataset/coco) image</figcaption>
+</figure>
+
+Object detection models require large amounts of annotated data to learn about objects of interest. Some commonly used
+datasets are: [MS COCO](https://paperswithcode.com/dataset/coco),
+[Pascal VOC](https://paperswithcode.com/dataset/pascal-voc),
+[Open Images](https://paperswithcode.com/dataset/open-images-v7),
+and [ImageNet](https://paperswithcode.com/dataset/imagenet).
+
+Object detection has a wide range of applications from self-driving cars, facial recognition, video surveillance,
+medical imaging, robotics, and more. Therefore, it is crucial to evaluate object detection models to understand their
+performance and applicability any real-world situation.
+
+## Evaluation Metrics for Object Detection Models
+
+Evaluation of object detection models requires ground truths and model inferences. The ground truths in an image are
+objects outlined by bounding boxes each labelled with a class, typically annotated by human labellers. The model
+inferences for an image are the labelled bounding boxes having confidence scores (the model's certainty of
+correctness), as if the model annotated the image.
+
+A [bounding box matcher](./geometry-matching.md) can align an image's ground truths and model inferences to produce
+[TP / FP / FN counts](./tp-fp-fn-tn.md). These counts are foundational for computing more detailed metrics, which
+provide insight into the model performance. Different evaluation configurations can be tuned to filter out model
+inferences before/after employing a matching algorithm such as filtering out inferences with a confidence score under
+0.01 and/or ignoring matches where the [IoU](./iou.md) (the overlap between the ground truth and and inference
+bounding box) under 0.5.
+
+Below are some of the commonly used metrics for evaluating object detection models:
+
+1. [**Precision**](./precision.md): Precision measures the ratio of correctly detected objects to all objects detected
+    by the model. High precision indicates a low rate of false positives.
+
+    $$\text{Precision} = \frac{\text{TP}}{\text{TP} + \text{FP}}$$
+
+2. [**Recall**](./recall.md): Recall measures the ratio of correctly detected objects to all actual objects (the
+    annotations). High recall indicates that the model is good at detecting most of the objects labelled by humans.
```

Comment:
`annotations` -> we've been using the terminology `ground truths` so far.  Is there a specific reason to call these `annotations`?

Reply:
I like the word annotations because it adds a hint of labelling